docs/topics/sealed-classes.md

```diff
@@ -50,6 +50,18 @@ Subclasses of sealed classes must have a proper qualified name. They can't be lo
 >
 {type="note"}
 
+### Inheritance in multiplatform projects
+
+In [multiplatform projects](mpp-intro.md), there comes one more restriction: direct subclasses of sealed classes must
+reside in the same source set. It applies to sealed classes without the [`expect`/`actual`](mpp-connect-to-apis.md) modifiers.
+
+A sealed class can be declared as `expect` in a common source set and have `actual` implementations in platform source sets.
```

Comment:
The second paragraph is a bit unclear for me — it seems to be a little out of touch with the beginning, where the restriction is explained.
I cannot offer a better structure here yet, but I think we can use this one (I mean yours) and return to it in the future. WDYT?

Reply:
I've changed the intro a bit. Not sure this is worth returning to, let's see if we have complaints.